roles/setup_replication/vars/EPAS_Debian.yml

```diff
@@ -3,6 +3,7 @@ pg_data: "/var/lib/edb-as/{{ pg_version }}/{{ pg_instance_name }}"
 pg_default_data: "/var/lib/edb-as/{{ pg_version }}/{{ pg_instance_name }}"
 pg_log: "/var/log/edb"
 pg_wal: ""
+pg_local_wal_archive_dir: ""
```

Comment:
Please define it in `defaults/main.yml`.

Reply:
Parameter removed and added to `defaults/main.yml`.